README.md

```diff
@@ -79,19 +79,19 @@ Pacdef will make sure you have the following packages installed for each package
 
 Note that the name of the section corresponds to the ecosystem it relates to, rather than the package manager it uses.
 
-
 ## Supported backends
 
 At the moment, supported backends are the following.
 Pull requests for additional backends are welcome!
 
-| Application | Package Manager | Section     | feature flag | Notes                                                                                                     |
-|-------------|-----------------|-------------|--------------|-----------------------------------------------------------------------------------------------------------|
-| Arch Linux  | `pacman`        | `[arch]`    | `arch`       | includes pacman-wrapping AUR helpers (configurable)                                                       |
-| Debian      | `apt`           | `[debian]`  | `debian`     | minimum supported apt-version 2.0.2 ([see upstream](https://gitlab.com/volian/rust-apt))                  |
-| Flatpak     | `flatpak`       | `[flatpak]` | built-in     | can manage either system-wide or per-user installation (configurable)                                     |
-| Python      | `pip`           | `[python]`  | built-in     |                                                                                                           |
-| Rust        | `cargo`         | `[rust]`    | built-in     |                                                                                                           |
+| Application | Package Manager | Section     | feature flag | Notes                                                                                    |
+| ----------- | --------------- | ----------- | ------------ | ---------------------------------------------------------------------------------------- |
+| Arch Linux  | `pacman`        | `[arch]`    | `arch`       | includes pacman-wrapping AUR helpers (configurable)                                      |
+| Debian      | `apt`           | `[debian]`  | `debian`     | minimum supported apt-version 2.0.2 ([see upstream](https://gitlab.com/volian/rust-apt)) |
+| Flatpak     | `flatpak`       | `[flatpak]` | built-in     | can manage either system-wide or per-user installation (configurable)                    |
+| Python      | `pip`           | `[python]`  | built-in     |                                                                                          |
+| Rust        | `cargo`         | `[rust]`    | built-in     |                                                                                          |
+| Rust        | `rustup`        | `[rustup]`  | built-in     | See the comments below about the syntax of the packages in the group file.               |
```

Comment:
1. Make the first column Rustup please to differentiate it from binaries built with rustc.
2. You can make the comment shorter and input a hyperlink to the rustup section in the README (https://github.com/InnocentZero/pacdef/tree/main?tab=readme-ov-file#rustup)

Reply:
Should be done now.